redfishMockupServer.py

```diff
@@ -694,15 +695,21 @@ def main():
         shortForm = args.short_form
         ssdpStart = args.ssdp
 
+        # If inside a Docker container, expose on all interfaces
+        if os.environ.get('INSIDE_DOCKER_CONTAINER', False):
```

Comment:
instead of using ENV and splitting logic between inside and ouside of docker, why you just always bind to 0.0.0.0 instead?

Reply:
Fixed; made a change to the Dockerfile to specify 0.0.0.0 to avoid having to do this.